main.go

```diff
@@ -175,7 +106,7 @@ func createStorageHandle(flags *flagStorage) (storageHandle storage.StorageHandl
 ////////////////////////////////////////////////////////////////////////
 
 // Mount the file system according to arguments in the supplied context.
-func mountWithArgs(
+func mountWithStorageHandle(
```

Comment:
dont rename this method. mountWithArgs sounds good

Reply:
@Tulsishah In this comment (https://github.com/GoogleCloudPlatform/gcsfuse/pull/1258/files#r1288264967), I meant to rename mountWithConn -> mountWithStorageHandle. Apologies if that was conveyed properly.

Reply:
Ohh got it thanks.